materialize-bigquery/transactor.go

```diff
@@ -247,7 +249,11 @@ func (t *transactor) Store(it *m.StoreIterator) (m.StartCommitFunc, error) {
 		var b = t.bindings[it.Binding]
 		b.storeFile.start()
 
-		converted, err := b.target.ConvertAll(it.Key, it.Values, it.RawJSON)
+		var flowDocument = it.RawJSON
+		if t.cfg.Advanced.HardDelete && it.Delete {
+			flowDocument = json.RawMessage("null")
```

Comment:
I think it's kind of confusing to have the sentinel value for issuing a deletion be the string `"null"`. I assume we're not doing `json.RawMessage(nil)` and checking for the actual `NULL` value here because the `flow_document` column is non-nullable - that seems alright, because it would be a pain to have a special case for that if `HardDelete` is enabled. Could we use a string value of `"delete"` instead for indicating that the row should be deleted?